src/Parallel/Algorithm.hpp

```diff
@@ -400,15 +400,14 @@ AlgorithmImpl<ParallelComponent, tmpl::list<PhaseDepActionListsPack...>>::
     : AlgorithmImpl() {
   (void)initialization_items;  // avoid potential compiler warnings if unused
   global_cache_ = global_cache_proxy.ckLocalBranch();
-  box_ = db::create<
-      db::AddSimpleTags<tmpl::flatten<
-          tmpl::list<Tags::GlobalCacheImpl<metavariables>,
+  box_ =
+      db::create<db::AddSimpleTags<tmpl::flatten<tmpl::list<
+                     Tags::GlobalCacheImpl<metavariables>,
```

Comment:
Can I request that you split the first commit "Add serialization for GlobalCache" into two commits, one for the whitespace changes and one for the "real" changes? There's a bit of noise in the diffs right now. I think the other commits are fine :)